src/Game.cpp

```diff
@@ -1,5 +1,21 @@
 #include "Game.hpp"
-
+#include<vector>
+#include<iostream>
+#include<algorithm>
+#include<ostream>
+#include<string>
```

Comment:
Wszystkie te includy poza stdexcept chyba są zbędne. Usuń je i jeśli się kompiluje to wkomituj. To już będzie wszystko poprawnie.